prow/plugins/trigger/pull-request_test.go

```diff
@@ -257,6 +257,14 @@ func TestHandlePullRequest(t *testing.T) {
 			ShouldBuild: false,
 			prAction:    github.PullRequestActionClosed,
 		},
+		{
+			name: "Trusted user labeled PR with ok-to-test should build",
```

Comment:
could you also test the cases when it should not build?

Reply:
Done

Reply:
I think we should also have a testcase for when the bot adds the label, because its kinda important that we do not trigger the tests here in that case?